backend/curriculum_tracking/tests/test_RecruitProject.py

```diff
@@ -82,3 +84,23 @@ def test_too_long(self):
         self.assertIn("-test-project-", repo_name)
         self.assertIn(self.user.first_name, repo_name)
         self.assertIn(self.user.last_name, repo_name)
+
+
+class get_total_duration(TestCase):
```

Comment:
This name is weird. Why?  Please take a look at the names of other test cases and see if you can pattern match.